src/app/components/ATIAnalytics/canonical/index.tsx

```diff
@@ -44,6 +39,21 @@ const addOperaMiniExtremeScript = (atiPageViewUrlString: string) => {
   );
 };
 
+const addLiteScript = (atiPageViewUrlString: string) => {
+  return (
+    <Helmet>
+      <script type="text/javascript">
+        {`
+          var xhr = new XMLHttpRequest();
+          xhr.open("GET", "${atiPageViewUrlString}", true);
+          xhr.withCredentials = true;
+          xhr.send();
+        `}
```

Comment:
Should we extract this into a separate file, like we did for the`sendBeaconOperaMiniScript`?

Reply:
Moved it into its own file 👍